src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let tempSum = 0;
+  let sum;
+
+  return function add(n) {
+    if (isNaN(n)) {
```

Comment:
I liked this approach. You can add `&& n === undefined` so `adder(2)('str')` doesn't return a value.